_source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md

```diff
@@ -335,10 +290,10 @@ now are the following two:
 
 - `cookie.httponly`: this option tells the browser that JavaScript code should
   not be allowed to access the session data. JavaScript on clients is a
-  dangerous thing, ensuring your cookies that contain identity information are
-  safe is always of top importance.
-- `secret`: this option should be a long random string that you create. It
-  should be the same across all your webservers, but never shared publicly or
+  dangerous thing. Ensuring your cookies that contain identity information are
```

Comment:
```suggestion
  dangerous thing. It's essential to ensure the safety of cookies that contain identity information.
```
this sentence is super convoluted, if you're going to edit, let's remove passive voice and awkward syntax.